src/Serialization/Groups.jl

```diff
@@ -157,7 +139,7 @@ end
 
 function load_object(s::DeserializerState, ::Type{PermGroup})
   n = load_object(s, Int, :degree)
-  generators = load_object(s, Vector, (Vector{Int}, Int), :gens)
+  generators = load_object(s, Vector{Vector{Int}}, :gens)
   G = permutation_group(n, [perm(x) for x in generators])
   load_attrs(s, G)
```

Comment:
```suggestion
```
this can go, right? And similarly the `save_attrs` 6 lines above

Reply:
yes